src/modules/commander/failure_detector/FailureDetector.cpp

```diff
@@ -47,28 +47,43 @@ FailureDetector::FailureDetector(ModuleParams *parent) :
 {
 }
 
-bool FailureDetector::resetStatus()
+bool FailureDetector::resetAttitudeStatus()
 {
-	bool status_changed = _status != FAILURE_NONE;
-	_status = FAILURE_NONE;
+
+	int attitude_fields_bitmask = _status & (FAILURE_ROLL | FAILURE_PITCH | FAILURE_ALT | FAILURE_EXT);
```

Comment:
@bresch / @bkueng  I could not come up with a better naming for this variable.. given that contains FAILURE_ALT and FAILURE_EXT is not really an attitude_fields_bitmask .. any suggestion?

Reply:
non_esc_bitmask? Otherwise this is fine.